### ISSUE TITLE
[Feature request] Option about Mark All as Read button position - iOS

### ISSUE THREAD

**Issue by celebrion** [Ranchero-Software/NetNewsWire#3145]
From an accessibility point-of-view it'd be great to give the user an option to move the Mark All as Read button to the right or the left.

**Comment by brentsimmons** [comment]
Would you explain more about how this would be good for accessibility?

**Comment by celebrion** [comment]
If you use your iPhone one-handed, you have to reach across the screen to hit the "mark all as read" button. Unless you have a big hand, this is hard.
Extra so since the app lacks the "pull up from bottom to mark all as read" as other RSS readers have.

**Comment by farginnoob** [comment]
Agree, especially since go to next unread article button is not super useful (since I can’t see what’s coming up).

**Comment by mattinatux** [comment]
As a potential resolution, perhaps centering the Mark All as Read button in the bottom bar would be helpful. 

I share this accessibility request. The current UX makes one handed use prohibitive when in the individual feed view.
